web/modules/custom/dpl_react_apps/src/Controller/DplReactAppsController.php

```diff
@@ -126,6 +127,15 @@ public static function authUrl(): string {
     );
   }
 
+  /**
+   * Builds an url for the react apps to for proxy urls.
+   */
+  public static function urlProxyUrl(): string {
+    return self::ensureUrlIsString(
+      Url::fromRoute('dpl_url_proxy.generate_url')->toString()
```

Comment:
Consider whether this creates a dependency on `dpl_url_proxy`.

Perhaps it is only a soft dependency if `url-proxy-url` is not required. In the current form it seems like a hard dependency.